npm/npm_import.bzl

```diff
@@ -787,3 +795,33 @@ def npm_import(
         bins = bins,
         npm_translate_lock_repo = npm_translate_lock_repo,
     )
+
+def list_patches(name, include_patterns = ["*.diff", "*.patch"], exclude_patterns = []):
+    """Write a file containing all patches in the current folder to the source tree.
```

Comment:
nit: Write a file containing a list of all patches in the current folder to the source tree.